### ISSUE TITLE
Multisig Backup PDF incorrect device type

### ISSUE THREAD

**Issue by tristanhcole** [cryptoadvance/specter-desktop#1669]
When I setup a multisig wallet using a ColdCard, Ledger Nano X and Trezor. The device type for the Ledger Nano X is being shown as a Trezor on the multisig backup PDF

**Comment by k9ert** [comment]
Unfortunately, i don't have a Ledger and can't test this. Would be cool if anyone else can test that and maybe confirm?

**Comment by tristanhcole** [comment]
It looks like Multisig is also misspelt as "Mutlisig" in the PDF document and file name

**Comment by thesimplekid** [comment]
I just tested this and was not able to reproduce.  I used a bitcoin core wallet a coldcard and a ledger and the device type off all of them in the pdf back up is correct. Multisig is also spelled correctly in the document.  The file name does not have multi sig in it, it is named name-of wallet_backup.pdf

![image](https://user-images.githubusercontent.com/8606367/164498964-b3ae295b-6e6b-4b55-9347-283d7cd37a20.png)
![image](https://user-images.githubusercontent.com/8606367/164499194-0056e38e-c6dc-4057-901d-f5e128e020fb.png)
